### PR TITLE
Tb/chore/validate content port author

### PR DESCRIPTION
**Pull request by tanvibhakta**
These are some quick fixes to content I made today.

Starting tomorrow I will use daily branches, so feel free to merge this one in.

### CODE REVIEW

File: src/templates/article.js [hunk 1]
```diff
@@ -104,9 +104,10 @@ export default ({ data }) => {
         />
       )}
       <div className="flex flex-col lg:w-2/3 self-end">
-        <Heading type="h1" className="text-gray-10 my-3 lg:my-4">
-          {article.title}
-        </Heading>
+        <Heading
+          className="text-gray-10 my-3 lg:my-4"
+          content={`<h1>${article.title}</h1>`}
```
Comment (meetbhatt23) [inline src/templates/article.js:109]: This seems like a hack. I know why you needed to do this but can we use this opportunity to add the ability to use html `tag` on the `Heading` component?

Comment (tanvibhakta) [reply src/templates/article.js:109]: Would you like me to prioritise this, or should I pick this up after I'm done with my current task?

Comment (tanvibhakta) [reply src/templates/article.js:109]: This makes sense to me, but perhaps as a separate PR as opposed to within this one?

Comment (meetbhatt23) [reply src/templates/article.js:109]: Can we add a `TODO` and table this for later?

Comment (tanvibhakta) [reply src/templates/article.js:109]: https://github.com/obvious/website-v2-source-wordpress/blob/master/src/components/Heading.js#L13

Yes, I have marked it as TODO a month ago.

**Comment by tanvibhakta** [comment]
Can this be merged now?

File: src/templates/article.js (hunk 1 above)
Comment (meetbhatt23) [reply src/templates/article.js:109]: Not that, the `tag` bit we have as a task on Linear as well. I meant this particular line that will need to be removed once the tag prop is in.

Comment (tanvibhakta) [reply src/templates/article.js:111]: Thanks for being explicit! 
I've made the change, as well as some other places this pops up. Now that I've done this, it about touches the rule of three, so I'll make sure to prioritize NUWEB-132 🙂

**Pull request merged**